xcell/mappers/__init__.py

```diff
@@ -1,8 +1,13 @@
 # flake8: noqa
 from .mapper_base import MapperBase
+from .mapper_base import MapperSDSS
```

Comment:
Just for cleanliness, can you put all the SDSS mappers together in this import list (and also get the eBOSS mappers together)